src/super_gradients/recipes/cityscapes_ddrnet.yaml

```diff
@@ -55,7 +55,17 @@ architecture: ddrnet_23
 
 training_hyperparams:
   max_epochs: 500
-  initial_lr: 0.0075   # batch size 24
+  initial_lr: # batch size 24
+    default: 0.075
```

Comment:
Maybe reverse the mapping to make it shorter? Thinks it's more clear this way:
```
initial_lr:
  default: 0.0075
  head: 0.075
```

Reply:
No head module for darned.
Going the other way around would be much longer.